config/local/discovery-service.yml

```diff
@@ -9,7 +9,7 @@ apiml:
         staticApiDefinitionsDirectories: config/local/api-defs
     security:
         ssl:
-            verifySslCertificatesOfServices: true
+            verifySslCertificatesOfServices: false
```

Comment:
Is there a reason to turn off the verification of the certificates?
The certificates for the local development are packaged in the repository